picoquic/frames.c

```diff
@@ -5263,15 +5263,15 @@ int picoquic_decode_frames(picoquic_cnx_t* cnx, picoquic_path_t * path_x, const
 {
     const uint8_t *bytes_max = bytes + bytes_maxsize;
     int ack_needed = 0;
-    int is_path_validating_packet = 1; /* Will be set to zero if non validating frame received */
+    int is_path_probing_packet = 1; /* Will be set to zero if non validating frame received */
```

Comment:
Maybe the comment should be updated as well.

(Thank you for your quick response.)